### ISSUE TITLE
Connect to nREPL session?

### ISSUE THREAD

**Issue by sparkofreason** [clojupyter/clojupyter#60]
Is it possible to connect to an existing nREPL session, as can be done with gorilla?

**Comment by didiercrunch** [comment]
There are no way to do that currently.  You can use https://github.com/clojupyter/lein-jupyter to use it with leningen but not directly with nREPL.

It would not be very hard to integrate but the user interface to provide is unclear to me.  I'll take a look at what gorilla does.

**Comment by sparkofreason** [comment]
Thanks - I had forgotten about lein-jupyter. That should get me going.

**Comment by Cognipacity** [comment]
Any progress on this? It'd be great to be able to use Cursive (IntelliJ) and clojupyter together via nREPL connection.

**Comment by behrica** [comment]
See comment at https://github.com/clojure-emacs/cider-nrepl/issues/572 on how to potentially change clojupyter to allow cider interop. Maybe some cider middleware for handling content types can be reused

**Comment by luojieshun** [comment]
IIRC, the middleware was added for convenience. It should be possible to
defer adding the mime annotations later in the handler of eval message.
(I think for now if you comment out the middleware, things should still
kind of works) however things like images will not be displayed correctly.

**Comment by behrica** [comment]
This was closed IMHO by
https://github.com/clojupyter/clojupyter/pull/79

**Comment by klausharbo** [comment]
Yes, it prints it the port address to `stdout`.  However, it can sometimes be incovenient to look there, so you can also get the port from within Jupyter by evaluating the form `(clojupyter.misc.nrepl-server/nrepl-server-addr)` which returns the IP address and port as a string like `"/127.0.0.1:57329"`.

**Comment by behrica** [comment]
> Yes, it prints it the port address to `stdout`. However, it can sometimes be incovenient to look there, so you can also get the port from within Jupyter by evaluating the form `(clojupyter.misc.nrepl-server/nrepl-server-addr)` which returns the IP address and port as a string like `"/127.0.0.1:57329"`.

The function  `(clojupyter.misc.nrepl-server/nrepl-server-addr)`  does not exist anymore.
I suggest to "println" the port and not only log it.


I have tried to connect to it from VSCode and it worked out of the box. So this can be very useful, so I suggest to "println" the port on startup.
